Makefile

```diff
@@ -12,18 +12,17 @@ IMPORT_PATH=$(IMPORT_HOST)/Jigsaw-Code/outline-apps
 all: android apple linux windows
 
 ROOT_PKG=client/src/tun2socks
-# Don't strip Android debug symbols so we can upload them to crash reporting tools.
-ANDROID_BUILD_CMD=$(GOBIND) -a -ldflags '-w' -target=android -androidapi 19 -tags android -work
 
 android: $(BUILDDIR)/android/tun2socks.aar
 
 $(BUILDDIR)/android/tun2socks.aar: $(GOMOBILE)
 	mkdir -p "$(BUILDDIR)/android"
-	$(ANDROID_BUILD_CMD) -o "$@" $(IMPORT_PATH)/$(ROOT_PKG)/outline/tun2socks $(IMPORT_PATH)/$(ROOT_PKG)/outline/shadowsocks
+  # Don't strip Android debug symbols so we can upload them to crash reporting tools.
```

Comment:
I think we are still stripping the debug symbols (`-w`)? Also I think we need to consider whether we should include debug symbols, that increase the binary size in a significant amount.

Reply:
-s is what disables the symbol table.
-w is disable DWARF generation

Ideally we would output tun2socks with debug info, and the client build would remove them. Is that possible?

But that's out of scope for this PR